protofire/polymesh-explorer
# Adding an Error Boundary

# Client error managing
## Fix:
- Avoid empty value on Autocomplete when click after delete characters
- Managing when venue is empty on Settlement Instruction Page

## Misc
- Use same background in all application to avoid contrasts when the main container is big.
- Open the footer link on a new page
- On Identity page the secondary keys selector open the Account details page when is selected
- On Account page is used `Venues` instead of `Venue`

## To test:
1. Go to search bar  and paste  `0x0400000000000000000000000000000000000000000000000000000000000000` into the search bar. Backspace to delete a character. Click on the displayed search result. Application should not break.

2. Go to search bar and paste the Instruction ID `12760`. Application should not display an error.
